sdk/eventhub/event-hubs/tests.yml

```diff
@@ -13,5 +13,3 @@ extends:
           Location: 'usgovarizona'
         China:
           Location: 'chinaeast'
-      EnvVars:
-        AZURE_LOG_LEVEL: info
```

Comment:
Surprising to me that this is the source - would we need to change https://github.com/maorleger/azure-sdk-for-js/blob/fix-backup-keyvault/sdk/eventhub/event-hubs/test/utils/logging.ts#L12 as well?

I only ask because (and I dont know this for sure) I expected keyvault's tests.yml to be used, not this one. But I am not at all confident in my understanding here 😄